style.css

```diff
@@ -1,24 +1,36 @@
 
+body  { 
+      margin: 0px;
```

Comment:
The indentation is messed up in this file. The reason for this is that for empty space at the begging of a line you are using combination of spaces and tabs. Spaces have fixed width, while tabs are relative and their representation is dependent from a program which you are using. So although everything looks ok on your machine, here it looks bad.

From now on, you should be using Sublime for programming, it is simple and has various plugins. Once you install it, set it up to use always spaces instead of tabs (menu in the bottom right corner)  and reformat this file.